fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/sound/sound.xml

```diff
@@ -84,7 +84,71 @@
         <Sound WwiseData="true" WwiseEvent="flaps380stop" NodeName="BASE_RIGHT" ConeHeading="180" CancelConeHeadingWhenInside="false" Continuous="false" LocalVar="A32NX_IS_FLAPS_MOVING" Units="BOOL" Index="1">
             <Range UpperBound="0" />
         </Sound>
-
+        
+        <!-- CABIN ANNOUNCEMENTS  ============================================================================================-->
+        <Sound WwiseEvent="custom_sound_01" WwiseData="false" FileName="WELCOME_ONBOARD" Continuous="False" LocalVar="A380X_WELCOME_ANC" Units="bool" MuteOnVPChange="True" ViewPoint="Inside">
+            <Requires LocalVar="A380X_WELCOME_ANC" Units="bool">
+                <Range LowerBound="1" UpperBound="1" />
+            </Requires>
+            <VolumeCurve LocalVar="A380X_RMP_1_PA_VOL" Units="Number" Index="1">
```

Comment:
The volume should come from whichever RMP is used by the pilot. That logic should be in https://github.com/flybywiresim/aircraft/blob/master/fbw-a380x/src/systems/systems-host/systems/Communications/SimAudioManager.ts

Reply:
Will Implement it. I figured out, that anyway the volume should be controlled by a custom L Var since the sound.xml doesnt allow to play the sounds while switching ViewPoints with out interrupting the playback